apps/src/code-studio/components/header/ProjectUpdatedAt.jsx

```diff
@@ -75,6 +82,10 @@ const styles = {
   container: {
     display: 'block',
     width: 160,
+    height: 16,
+  },
+  floatRight: {
```

Comment:
this should always allow you not to add floatRight probably. (Though this suggestion relies on the idea that css can read `html[dir]` attribute, which normally it can)